canonical-web-and-design/anbox-cloud.io
# Create basic page stub for the demo

## Done

Created a basic layout for the demo page. No need for design yet, we'll finesse the page once the content is in place.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/demo
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #46
